src/apis/pp_console_ws_worker.erl

```diff
@@ -52,7 +52,8 @@
 -define(HEADER_JSON, {<<"Content-Type">>, <<"application/json">>}).
 
 %% Channels
--define(ORGANIZATION, <<"organization:all">>).
+-define(ORGANIZATION_CHANNEL, <<"organization:all">>).
```

Comment:
Being picky here but as they are used as topics could we rename `_CHANNEL` to `_TOPIC`?

Reply:
Absolutely! I'm a little surprised this never occurred to me as I was typing everything.